helm-charts/strimzi-kafka-operator/templates/_kafka_image_map.tpl

```diff
@@ -13,6 +13,10 @@
               value: {{ default .Values.tlsSidecarZookeeper.image.repository .Values.imageRepositoryOverride }}/{{ .Values.tlsSidecarZookeeper.image.name }}:{{ default .Values.tlsSidecarZookeeper.image.tagPrefix .Values.imageTagOverride }}-kafka-2.4.0
             - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
               value: {{ default .Values.kafkaExporter.image.repository .Values.imageRepositoryOverride }}/{{ .Values.kafkaExporter.image.name }}:{{ default .Values.kafkaExporter.image.tagPrefix .Values.imageTagOverride }}-kafka-2.4.0
+            - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
```

Comment:
This is just a general comment. Do we want to give user's the option to use their own CC image?

Reply:
It would be easier and more secure if we didn't give users the option. I don't think people would need/want a custom CC image anyway, I feel like it would default the purpose of the integration. Anyways, I am very open to removing the respective env vars from the CO deployment simply to reduce complexity.